docs/2.7/index.html

```diff
@@ -32,7 +32,7 @@ <h1 class="site-title"><a class="title-link" href="https://svgjs.com/docs/2.7">S
 
     
 <aside class="sidebar-menu">
-  <p class="intro">Find docs for versions 3.0 <a href="../3.0">here</a></p>
```

Comment:
I incorporated this change manually because i didnt recompile the docs. It was a hack because the docs are not where are the supposed to be yet. Can you maybe put this one back?

Reply:
Thanks for spotting that.  I put it back manually.

Reply:
Awesome!! Than you!